microshift_running_apps/microshift-operators.adoc

```diff
@@ -0,0 +1,13 @@
+:_content-type: ASSEMBLY
+[id="operators-with-microshift"]
+= Using Operators
+include::_attributes/attributes-microshift.adoc[]
+:context: operators-microshift
+
+toc::[]
+
+Operators integrate with Kubernetes APIs and CLI tools such as `kubectl` and `oc` commands. They provide the means to manage applications and the resources used by applications, such as deploying a database or message bus.
+
+Operators offer a more granular configuration experience. You configure each component by modifying the API that the Operator exposes instead of modifying a global configuration file. Even though {product-title} applications are often deployed in static environments, you can install Operators with manifests for your use.
```

Comment:
```suggestion
Operators offer a more granular configuration experience. You configure each component by modifying the API that the Operator exposes instead of modifying a global configuration file. Even though {product-title} applications are often deployed in static environments, you can install Operators with manifests.
```
Minimalism suggestion. It might be worth explaining how manifests are different than static environments - it isn't clear to me what the distinction is.

Reply:
I expanded this a bit. So what's happening here are two things: 1) static environments don't really need operators -- 'set and forget' is the general direction for applications running on MicroShift. And two, in OCP, Operators are installed and used through OLM (the Operator Lifecycle Manager). Since this is a very lightweight Kubernetes, using OLM isn't an option. You can install the Operators you want with manifests, though, if you need them.